pom.xml

```diff
@@ -0,0 +1,203 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                      http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>io.xspec</groupId>
+  <artifactId>xspec</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  
+  <name>XSpec implementation</name>
+  <description>A unit test framework for Xsl, XQuery and Schematron</description>
+  <inceptionYear>2008</inceptionYear>
+  <url>https://github.com/xspec/xspec</url>
+  
+  <licenses>
+    <license>
+      <name>The MIT License</name>
+      <distribution>repo | manually</distribution>
+      <url>https://opensource.org/licenses/MIT</url>
+      <comments>Copyright (c) 2008-2017 Jeni Tennison</comments>
+    </license>
+  </licenses>
+  
+  <developers>
+    <developer>
+      <name>Jeni Tennison</name>
+      <email>jeni.@jenitennison.com</email>
+      <roles>
+        <role>creator</role>
+      </roles>
+    </developer>
+    <developer>
+      <name>Sandro Cirulli</name>
+      <email>sandro.cirulli@oup.com</email>
+      <roles>
+        <role>current maintainer</role>
+      </roles>
+    </developer>
+    <developer>
+      <name>Florent George</name>
+      <email>info@fgeorges.org</email>
```

Comment:
Florent gave his consent but asked to use this email address:

```
 fgeorges@fgeorges.org
```

Reply:
You are right, pull request is not up to date